aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -285,18 +286,33 @@ Tensor _s_where(const Tensor& condition, const Tensor& self, const Tensor& other
   return ret;
 }
 
+std::tuple<Tensor, Tensor> _mode(const Tensor& self, int64_t dim, bool keepdim) {
+  return at::native::mode(self, dim, keepdim);
```

Comment:
You can just drop the `_mode` operator. Since it starts with an underscore, users shouldn't be relying on it.  Just add an exception to the backward compatibility check:
https://github.com/pytorch/pytorch/blob/d22f700f9ea566f07ad392b40889aae1697303a6/test/backward_compatibility/check_backward_compatibility.py#L75